usefulUtilities/airAndWhiteboardDraw/entityResources/entity/whiteboard.json

```diff
@@ -0,0 +1,638 @@
+    "queryAACube": {
+        "x": -71.7729721069336,
+        "y": 890.8225708007812,
+        "z": -7.1639909744262695,
+        "scale": 0.1517527550458908
+    },
+    "grab": {
+        "grabbable": false,
+        "equippableLeftRotation": {
+            "x": -0.0000152587890625,
+            "y": -0.0000152587890625,
+            "z": -0.0000152587890625,
+            "w": 1
+        },
+        "equippableRightRotation": {
+            "x": -0.0000152587890625,
+            "y": -0.0000152587890625,
+            "z": -0.0000152587890625,
+            "w": 1
+        }
+    },
+    "damping": 0,
+    "angularDamping": 0,
+    "collisionMask": 7,
+    "collidesWith": "static,dynamic,kinematic,",
+    "script": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/whiteboardReset.js",
+    "scriptTimestamp": 1554489729490,
+    "text": "Reset",
+    "lineHeight": 0.05999999865889549,
+    "clientOnly": false,
+    "avatarEntity": false,
+    "localEntity": false,
+    "faceCamera": false,
+    "isFacingAvatar": false
+},
+    {
+    "id": "{7535a63a-1d80-47b5-844a-4d11514326a9}",
+    "type": "Box",
+    "lastEdited": 1554923712547407,
+    "parentID": "{f8b95e5a-a886-4606-b023-82e0e026d33f}",
+    "name": "Whiteboard Palette Square",
+    "locked": true,
+    "userData": "{\"textureURL\":\"http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/usefulUtilities/airAndWhiteboardDraw/entityResources/resources/textures/cyan.png\"}",
+    "position": {
+        "x": 0.7458062171936035,
+        "y": -0.5976181030273438,
+        "z": -0.0008983612060546875
+    },
+    "dimensions": {
+        "x": 0.05000000074505806,
+        "y": 0.05000000074505806,
+        "z": 0.0010000000474974513
+    },
+    "rotation": {
+        "x": -0.0003204345703125,
+        "y": 0.0000152587890625,
+        "z": -0.0001678466796875,
+        "w": -1
+    },
+    "created": 1554419713257669,
+    "lastEditedBy": "{9e99801d-9ccb-4f48-9f2d-54abf0130e7d}",
+    "queryAACube": {
+        "x": -71.73727416992188,
+        "y": 890.8565673828125,
+        "z": -6.885195255279541,
+        "scale": 0.07071775197982788
+    },
+    "grab": {
+        "grabbable": false,
+        "equippableLeftRotation": {
+            "x": -0.0000152587890625,
+            "y": -0.0000152587890625,
+            "z": -0.0000152587890625,
+            "w": 1
+        },
+        "equippableRightRotation": {
+            "x": -0.0000152587890625,
+            "y": -0.0000152587890625,
+            "z": -0.0000152587890625,
+            "w": 1
+        }
+    },
+    "damping": 0,
+    "angularDamping": 0,
+    "collisionMask": 7,
+    "collidesWith": "static,dynamic,kinematic,",
+    "script": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/drawSphereSpawnerClient.js",
```

Comment:
please replace this link with a relative link (which you'll then have to update upon release)

(i won't make this comment any more on this file, but please replace all absolute links with relative links)